apps/design-land/src/app/radio/examples/generated-radio-set/generated_radio_set.component.html

```diff
@@ -0,0 +1,7 @@
+<h2>RadioSet generated from a list</h2>
```

Comment:
Is an example for this needed? Generating a radio set from a list with an array isn't a UI thing that needs to be outlined? We already have the example of what a radio-set is.